packages/flame_bloc/LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
symlink

Reply:
we are not doing the symlink anymore because `pub publish` fails on linux, but thanks for pointing out, I need to add a proper file here

Reply:
I thought our publish script just replaced the links in time for the release